Source/monster.cpp

```diff
@@ -4574,18 +4561,13 @@ void SyncMonsterAnim(Monster &monster)
 	case MonsterMode::Charge:
 		graphic = MonsterGraphic::Attack;
 		monster.AnimInfo.CurrentFrame = 1;
-		monster.AnimInfo.NumberOfFrames = monster.MType->GetAnimData(MonsterGraphic::Attack).Frames;
 		break;
 	default:
 		monster.AnimInfo.CurrentFrame = 1;
-		monster.AnimInfo.NumberOfFrames = monster.MType->GetAnimData(MonsterGraphic::Stand).Frames;

```

Comment:
`Monster::ChangeAnimationData` will update `AnimInfo.NumberOfFrames` if necessary so I didn't see a reason to duplicate that logic here.